src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/Negotiation.java

```diff
@@ -156,4 +177,11 @@ public Set<Resource> getResources() {
         .flatMap(request -> request.getResources().stream())
         .collect(Collectors.toUnmodifiableSet());
   }
+
+  public Resource lookupResource(Set<Resource> resources, String resourceId) {
```

Comment:
please refactor. This method is not needed I believe or at least should not be public.

Reply:
Fixed in 673d2d0e5aa92b06fda147cf34949dd7444d4de4